src/main/java/net/dv8tion/jda/core/managers/GuildController.java

```diff
@@ -603,6 +605,8 @@ protected void handleResponse(Response response, Request<Void> request)
 
         Checks.notNegative(delDays, "Deletion Days");
 
+        Checks.check(delDays <= 7, "Deletion Days may not be bigger than 7.");
```

Comment:
Is the correct phrasing "may not" or "must not" here? Or are both valid?

Reply:
_Technically_ they're the same in general conversation, however if we abide by [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt), then we `must` is more correct.

Reply:
We don't really abide to that RFC within our documentation

Reply:
Since may could imply a scenario where it is acceptable to use a number above 7, I've decided to change the phrasing to must (abiding to RFC 2119).